src/libraries/Microsoft.Extensions.FileProviders.Physical/tests/AssemblyInfo.cs

```diff
@@ -0,0 +1,8 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Would you need to add the this file to be compiled in `src/libraries/Microsoft.Extensions.FileProviders.Physical/tests/Microsoft.Extensions.FileProviders.Physical.Tests.csproj`?

Reply:
Not in this one.  It's included automatically.